Gemfile

```diff
@@ -22,7 +22,7 @@ gemspec path: "swift"
 gemspec path: "terraform"
 
 # Sorbet
-gem "sorbet", "0.5.11156", group: :development
+gem "sorbet", "0.5.11178", group: :development
```

Comment:
This upgrade is necessary because `sorbet` version `0.5.11158` shipped with more accurate types for `Gem::Version`[^1]. Specifically, Dependabot uses `String`, `Gem::Version` and `Integer` interchangeably throughout the codebase.

[^1]: https://github.com/sorbet/sorbet/commit/8e8ac5509099153963e38c3c94b8a3ea27515df5